src/cases/containers/case-hearings/case-hearings.component.html

```diff
@@ -1,29 +1,40 @@
-<div class="govuk-tabs__panel" id="event">
-  <br>
-  <a *ngIf="hasRequestAction"
-     href="javascript:void(0)"
-     role="button" draggable="false"
-     class="govuk-button
-     govuk-button--start"
-     data-module="govuk-button"
-     (click)="createHearingRequest()"
-     style="padding-right: 10px;background-image:none;">
-    Request a hearing
-    <svg style="padding-left:5px;vertical-align: middle;" class="govuk-button__start-icon"
-         xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40"
-         focusable="false">
-      <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z"/>
-    </svg>
-  </a>
-  <exui-case-hearings-list
-    [status]="upcomingStatus"
-    [hearingList$]="upcomingHearings$"
-    [actions]="hearingsActions">
-  </exui-case-hearings-list>
-
-  <exui-case-hearings-list
-    [status]="pastAndCancelledStatus"
-    [hearingList$]="pastAndCancelledHearings$"
-    [actions]="hearingsActions">
-  </exui-case-hearings-list>
+<div class="govuk-tabs__panel">
+  <div class="govuk-width-container" *ngIf="serverError">
+    <div class="govuk-main-wrapper">
+        <p class="govuk-body">{{serverError.message}}</p>
+        <span class="govuk-body align-left">
+          <a href="javascript:void(0)" [routerLink]="['/', 'cases', 'case-details', caseId, 'hearings']"
```

Comment:
it seems angular will do nothing to navigate to the same page. so we need to use (click)="reload()"

on the reload() dispatch an action
this.hearingStore.dispatch(new fromHearingStore.LoadAllHearings(this.caseId));